examples/kustomize/argocd/kustomization.yaml

```diff
@@ -1,24 +1,24 @@
-# kustomize build . >argocd-deploy.yml
+# kubectl kustomize . > argocd-deploy.yaml
 #
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 namespace: argocd
 
 resources:
-- https://raw.githubusercontent.com/argoproj/argo-cd/v1.8.4/manifests/ha/install.yaml
-- argocd-vault-replacer-secret.yml
-- serviceaccount.yml
+- https://raw.githubusercontent.com/argoproj/argo-cd/v2.5.5/manifests/ha/install.yaml
```

Comment:
while you're in this file, do you mind changing this line to the following?

```
- github.com/argoproj/argo-cd/manifests/ha/cluster-install?ref=v2.5.5
```

Reply:
Done